src/client/components/Carousel/Carousel.component.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import './Carousel.style.css';
+
+const Carousel = ({ imageArray }) => {
+  const show = 3;
+  const [current, setCurrent] = useState(0);
+  const imageArraLength = imageArray.length;
+  const prevSlide = () => {
+    setCurrent(current === 0 ? imageArraLength - 1 : current - 1);
+  };
+  const nextSlide = () => {
+    setCurrent(current === imageArraLength - 1 ? 0 : current + 1);
+  };
```

Comment:
You've found a good and simple way to build a carousel using the concepts we've been teaching in the React class. That's good.

The end result is not as user friendly as it could be, however. It is fairly slow to update. You might end up seeing the same picture in the carousel twice while it is updating and it would be very hard to make the carousel animated if you wanted to achieve that.

I think you've achieved the goal for this task and therefore I'm approving the PR, but here is a suggestion how it could be improved in the future. But I think we have plenty of work, so merge it as it is and we can see if we find time to improve later.

1. Instead of using `useState` to determined what images should be rendered, render all of the images in the carousel. This way they are always ready for being displayed and you will not have to turn one image off and turn another one off to change the state of the carousel.
2. Use CSS to display and hide elements outside the "visible bounds" of the carousel.
3. Use CSS to manipulate positioning of elements in the carousel.
4. Optionally add CSS transitions to the positioning to make the carousel animate.